CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing Guidelines
+
+First, thank you for using VGG! We welcome for issues, pull requests, or asking/answering questions on [Discard channel](#contact-us). Before that, please read [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Create an Issue
+If you find any bug or have any idea, no matter a typo, a document error, or a runtime bug, feel free to create an issue.
+
+If it's a simple bug easy to fix, you can create a [pull request](#pull-request) directly, bypassing this step.
+
+1. Check [issue list](https://github.com/verygoodgraphics/vgg_runtime/issues) to avoid duplicated issues.
+2. If it's security related, please report to [VGG](mailto:bd@verygoodgraphics.com) directly.
+3. Follow the issue template, and fill in information as much as you can.
+
+We will reply ASAP.
+
+## Solve an Issue
+If you want to solve an unassigned issue in [#help-wanted](https://github.com/verygoodgraphics/vgg_runtime/labels/help%20wanted), you can reply under the comment thread, waiting for an assignment. You can create a [pull request](#pull-request) when the issue is sovled.
+
+## Pull Request
+If you can fix any bug or make any improvement, welcome to send a pull request. Both core team members and external contributors send pull requests which go through the same review process. Checkout [GitHub's pull request tutorial](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests) in case of any issue.
+
+When opening a pull request, you will be asked to sign a Contributor License Agreement (CLA).
+
+### basic guildlines
+- Keep each commit atomic, and commit with clear messages.
+- Follow the PR template, and write clear descriptions.
+
+### extra guildines for code
```

Comment:
typo & grammar problem. `Extra Guidelines for coding`

Reply:
fixed